### PR TITLE
Book page

### PR DESCRIPTION
**Pull request by allurivamsi5**
I have completed book_page .

### CODE REVIEW

File: create.py [hunk 1]
```diff
@@ -80,4 +82,12 @@ def logout():
...
     except:
         var1 = "You must first log in to logout"
-        return render_template("reg.html",var1 = var1)
\ No newline at end of file
+        return render_template("reg.html",var1 = var1)
+
+
+@app.route('/books/<id>')
+def books(id):
+    result = db.session.query(Books).filter(Books.isbn == id)
```
Comment (ashishk144) [inline create.py]: check if the user is logged in, if not do not display the page.

Comment (ashishk144) [inline create.py]: db.session.query(Books).filter(Books.isbn == id) returns an object and not a list. How are you rendering it on page?

File: templates/Book_Page.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html>
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
```
Comment (ashishk144) [inline templates/Book_Page.html]: extend from layout file

File: templates/Book_Page.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+      <style>
```
Comment (ashishk144) [inline templates/Book_Page.html]: use style sheets.

**Review by ashishk144 — COMMENTED** [review]
Work on the feedback

File: create.py (hunk 1 above)
Comment (allurivamsi5) [reply create.py]: Updated, checking user login.

Comment (allurivamsi5) [reply create.py]: Changed query of result

File: templates/Book_Page.html (hunk 2 above)
Comment (allurivamsi5) [reply templates/Book_Page.html]: Extending layout file is done...

File: templates/Book_Page.html (hunk 3 above)
Comment (allurivamsi5) [reply templates/Book_Page.html]: Style sheet also done.

**Comment by ashishk144** [comment]
Looks good to merge.

**Pull request merged**